djaoapp/templates/rules/app_dashboard.html

```diff
@@ -192,34 +192,34 @@ <h5 class="modal-title">Encrypted Session Key</h5>
                 <div class="controls">
                   <input class="form-control" v-model="sessionKey"
                          name="key" maxlength="32" type="text" disabled
-                         value="Generating ..." />
+                         value="{% trans %}Generating ...{% endtrans %}" />
```

Comment:
This one is tricky. It might be used in a conditional statement in the Javascript. to check.